src/resources/Enums.ts

```diff
@@ -844,6 +844,42 @@ export enum LicenseSection {
     searchapi = 'searchapi',
 }
 
+export enum SearchHubRawMetrics {
+    normalQueries = 'normalQueries',
+    recommendationQueries = 'recommendationQueries',
+    commerceProductListingQueries = 'commerceProductListingQueries',
+    users = 'users',
+    staticQueries = 'staticQueries',
+}
+
+export enum SearchHubLicenseMetrics {
```

Comment:
Where is this used?

Reply:
It is currently used in the Search hub section (consumption dashboard) in the Settings panel. This information is already available in the Admin-UI, but I think it would make more sense to expose it directly in Platform-client (though?)

Reply:
It's fine then 👍🏻